homeassistant/components/sector_alarm/alarm_control_panel.py

```diff
@@ -0,0 +1,82 @@
+import logging
+from homeassistant.components.sector_alarm import DOMAIN as SECTOR_DOMAIN
+import homeassistant.components.alarm_control_panel as alarm
+from homeassistant.const import (STATE_ALARM_DISARMED, STATE_ALARM_ARMED_HOME,
+                                 STATE_ALARM_ARMED_AWAY, STATE_ALARM_PENDING)
+
+import homeassistant.components.sector_alarm as sector_alarm
+
+DEPENDENCIES = ['sector_alarm']
+
+DOMAIN = 'sector_alarm'
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_platform(hass, config, async_add_entities,
+                               discovery_info=None):
+    """Initial setup of the platform."""
+    sector_connection = hass.data.get(SECTOR_DOMAIN)
```

Comment:
What happens here if no SECTOR_DOMAIN exist?

Reply:
Add a guard clause here that checks if discovery_info is None and return if so.